xml2rfc/parser.py

```diff
@@ -55,11 +55,7 @@ def __init__(self, cache_path=None, library_dirs=None, source=None,
                  templates_path=base.default_options.template_dir, verbose=None, quiet=None,
                  no_network=None, network_locs= [
                      'https://bib.ietf.org/public/rfc/',
-                     'https://xml2rfc.ietf.org/public/rfc/',
-                     'https://xml2rfc.tools.ietf.org/public/rfc/',
                      'http://bib.ietf.org/public/rfc/',
```

Comment:
Why look here? If the object exists the https: variant will return it. If it doesn't we just get 404's twice.

Reply:
Yeah, because `http://bib.ietf.org/public/rfc/` redirects to `https` it doesn't make sense to have it here. I'll remove that too.